commcare_export/writers.py

```diff
@@ -358,7 +358,7 @@ def best_type_for(self, val):
             elif self.is_mssql:
                 return sqlalchemy.NVARCHAR(collation=self.collation)
             if self.is_oracle:
-                return sqlalchemy.UnicodeText(collation=self.collation)
+                return sqlalchemy.Unicode(4000, collation=self.collation)
```

Comment:
shouldn't this be NVARCHAR(4000? Just looking at the sqlalchemy one above...)

Reply:
According to sqlalchemy, `Unicode` is converted under the hood on oracle to `VARCHAR2`, just as `UnicodeText` was converted to `CLOB` https://docs.sqlalchemy.org/en/13/dialects/oracle.html#unicode-specific-column-datatypes. I don't have a good sense of what is the preferred way to specify that, but this seems a bit more flexible and inline with our other usages.